core/trie/proof.go

```diff
@@ -134,153 +165,6 @@ func transformNode(tri *Trie, parentKey *Key, sNode StorageNode) (*Edge, *Binary
 	return edge, binary, nil
 }
 
-// pathSplitOccurredCheck checks if there happens at most one split in the merged path
```

Comment:
Note: This functions are reported unused by linter, hence I deleted them.
Maybe further PR #2252 will bring them back.